database_schemas/cassandra/benchflow.cql

```diff
@@ -23,14 +23,26 @@ CREATE TABLE database_sizes (
   trial_id text,
   dbms text,
   database_name text,
-  database_table text,
   size bigint,
   PRIMARY KEY ((trial_id, experiment_id), dbms)
 );
 
+CREATE TABLE io_data (
+  experiment_id text,
+  trial_id text,
+  container_properties_id uuid,
+  device text,
+  writes bigint,
+  reads bigint,
+  sync bigint,
+  async bigint,
+  total bigint,
+  PRIMARY KEY ((trial_id, experiment_id), container_properties_id)
+);
+
 CREATE TABLE environment_data (
   environment_data_id uuid,
-  container_properties_id uuid,
+  container_id text,
```

Comment:
We still might have a conflict, and it is not affordable.

Why is it difficult?

- You get the ID generated by Docker and pass it to the transformer
- The transformer uses it to get the right file, with the `container_properties` data
- You store the container properties in the `container_properties` data and get back the `uuid` generated for/from Cassandra
- You then use this new generated `uuid` for everything else you are already doing in the transformer
- You submit the analysers on each of the generated `uuid`s or one analyser working on all the containers for a given experiment and/or trial.